### PR TITLE
chore: update test instructions

### PR DESCRIPTION
**Pull request by Startouf**
[REDACTED]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,41 @@
...
+You can start by writing a `main.rb` (feel free to change this name) that will perform the conversion when running `ruby main.rb [input_dir] [output_dir]` of all the files of the input directory into the output dir
+
+## Test rating criteria
 
-### Test criteria
 - clean
 - extensible
 - robust (don't overlook edge cases, use exceptions where needed, ...)
-- tested
+- tested (based on the schema)
```
Comment (WalaThabet) [inline README.md:35]: j'ai pas compris la modif ici

Comment (Startouf) [reply README.md:35]: ah zut, je voulais pas pusher ça, je retire
